grt-eval.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+################################################################################
+#
+# This script runs Defects4j computation in parallel. It handles setup (excluding
```

Comment:
Please be more specific about what is computed.
It creates Table IV; does it do more?